skysprites/pubspec.yaml

```diff
@@ -6,3 +6,6 @@ homepage: https://github.com/domokit/sky_engine/tree/master/sky/packages/sky
 dependencies:
   sky: ">=0.0.36 < 0.1.0"
   sky_tools: ">=0.0.10 < 0.1.0"
+dependency_overrides:
+  sky:
+    path: ../sky/packages/sky
```

Comment:
Yes, that's how we have all the other packages set up in this repo.  Otherwise it will use the stable version of the sky package.